tampermonkey/PPT_Auth_AutoComplete.js

```diff
@@ -17,11 +17,8 @@
     document.getElementById("affinityGroupSelect").selectedIndex = 1;
 
     document.getElementsByName("enrolment[0].name")[0].value = "HMRC-PPT-ORG";
-    document.getElementById("input-0-0-name").value = "UTR";
```

Comment:
Should we remove UTR entry from the acceptance tests at this point too? I can do it as part of the ticket I'm working on if needed.

Reply:
Yes, that is exactly what I am doing here. 
You just picked a removed line :wink: 
Please confirm that.

Reply:
Sorry, I mean the log in function in the acceptance tests as in here 😬: https://github.com/hmrc/plastic-packaging-tax-acceptance/blob/master/src/test/scala/uk/gov/hmrc/test/ui/pages/helpers/AuthWizard.scala#L24